pinc/misc.inc

```diff
@@ -10,18 +10,6 @@
 
 use voku\helper\UTF8;
 
-/**
- * Return $arr[$key], or if it's not defined, $default.
- */
-function array_get($arr, $key, $default)
```

Comment:
We should not remove this function yet because it's used by code that isn't checked into git (ie: `noncvs` code) and perhaps others who use our code.

Instead, we should add typing for the bits we know (`array $arr`), set it as `@deprecated`, and do a `trigger_error("array_get() is deprecated, use the null coalescing operator instead", E_USER_DEPRECATED);`.